src/parser/main.js

```
@@ -1,9 +1,38 @@
+        ['\\$', '$'],    // Dollar sign replacement
+        ['\\"', '"'],    // Double quote replacement
+        ['\\`', '`'],    // Backtick replacement
+        ['\\\n', ''],    // Backslash newline (line continuation) replaced with nothing
+        ['\\\\', '\\']   // Double backslash replacement (done last)
+    ]);
+}
+
+function replace(str, replacements) {
+    let result = str;
+
+    for (let pair of replacements) {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'pair' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)